packaging/nnstreamer.spec

```diff
@@ -484,7 +487,6 @@ popd
 # 'lcov' generates the date format with UTC time zone by default. Let's replace UTC with KST.
 # If you ccan get a root privilege, run ln -sf /usr/share/zoneinfo/Asia/Seoul /etc/localtime
 TZ='Asia/Seoul'; export TZ
-$(pwd)/tests/unittestcoverage.py module $(pwd)/gst $(pwd)/ext %testtarget
```

Comment:
if unittestcoverage.py is unnecessary, how about removing this file?

Reply:
Yes but I am not sure it's okay for me to remove this...

Reply:
You may remove it if it is not used by .spec file.

Reply:
Got it. I will update the PR